index.js

```diff
@@ -16,10 +15,16 @@ Object.defineProperty(exports, 'mapFileCommentRegex', {
   }
 });
 
-
-function decodeBase64(base64) {
-  return (SafeBuffer.Buffer.from(base64, 'base64') || "").toString();
-}
+var decodeBase64 = typeof Buffer !== 'undefined' ? Buffer.from ?
+  function decodeBase64(base64) {
+    return Buffer.from(base64, 'base64').toString();
+  } :
+  function decodeBase64(base64) {
+    return new Buffer(base64, 'base64').toString();
+  } :
```

Comment:
```suggestion
  function decodeBase64(base64) {
    if (typeof value === 'number') {
      throw new TypeError('The value to decode must not be of type number.)
    }
    return new Buffer(base64, 'base64').toString();
  } :
```

[safer-buffer](https://github.com/ChALkeR/safer-buffer/blob/master/safer.js#L29-L31) checks to make sure that the value isn't a number (I believe that is because `new Buffer` allocates bytes of length `value` if the value is a number).